packages/forms/backend/sirius-components-collaborative-widget-reference/src/main/java/org/eclipse/sirius/components/collaborative/widget/reference/datafetchers/MutationAddReferenceValuesDataFetcher.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.widget.reference.datafetchers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
+
+import org.eclipse.sirius.components.annotations.spring.graphql.MutationDataFetcher;
+import org.eclipse.sirius.components.collaborative.widget.reference.dto.AddReferenceValuesInput;
+import org.eclipse.sirius.components.core.api.IPayload;
+import org.eclipse.sirius.components.graphql.api.IDataFetcherWithFieldCoordinates;
+import org.eclipse.sirius.components.graphql.api.IEditingContextDispatcher;
+import org.eclipse.sirius.components.graphql.api.IExceptionWrapper;
+
+import graphql.schema.DataFetchingEnvironment;
+
+/**
+ * The data fetcher used to add reference values.
+ *
+ * @author pcdavid
+ */
+@MutationDataFetcher(type = "Mutation", field = "addReferenceValues")
+public class MutationAddReferenceValuesDataFetcher implements IDataFetcherWithFieldCoordinates<CompletableFuture<IPayload>> {
```

Comment:
Why add datafetchers in the collaborative part of the project? Same comment for all the data fetchers